addons/hr_recruitment/models/hr_applicant_refuse_reason.py

```diff
@@ -7,7 +7,9 @@
 class ApplicantRefuseReason(models.Model):
     _name = "hr.applicant.refuse.reason"
     _description = 'Refuse Reason of Applicant'
+    _order = 'sequence'
 
+    sequence = fields.Integer(copy=False)
```

Comment:
Add a default value, otherwise it will be null and can mess up stuff up

Reply:
Now I have set the default value 1. OK for you?

Reply:
@ppr-odoo i think, normally the default value set for sequence field is 10.
thanks

Reply:
> @ppr-odoo i think, normally the default value set for sequence field is 10. thanks

10 is not fixed. I have asked  @kba-odoo. [link](https://github.com/odoo/odoo/blob/16.0/addons/im_livechat/models/chatbot_script_answer.py#L16)